ui/cypress/fixtures/stats.json

```diff
@@ -0,0 +1,5 @@
+{
+  "anonymousUsageStats":true,
```

Comment:
How about using mirage for this?

Reply:
I didn't know about MirageJS. I did a research and I found that there are two popular API mocking tools:
- Mirage JS
- Mock Service Worker (msw)

Here is a comparison of both: https://miragejs.com/docs/comparison-with-other-tools/#msw

And after reading different opinions about these two...

_"the biggest difference today is that MSW runs in a Service Worker which enables real-enough network requests that show up in the Network tab of the Devtools"_,

**but**

_"in Mirage JS, the in-memory database + ORM help maintain your data's relational integrity across reads and writes to all endpoints"_

(extracted from [here](https://github.com/miragejs/miragejs/issues/449#issuecomment-622519413))

So, I'm not sure which one could be the right one for the meshery project. What do you think?

Reply:
I guess we will move to MirageJS, but considering we have seldom of UI tests it will be worth to move away when Cypress fixes becomes redundant and less efficient. For now, we can use the cypress stubs & fixes as is you have implemented in this PR. 

PS: That's quite neat and beautiful work you have done here 😄